pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -62,16 +78,7 @@ type ExplainersConfig struct {
 	AlibiExplainer ExplainerConfig `json:"alibi,omitempty"`
 }
 
-func getExplainerHandler(modelSpec *ExplainerSpec) Explainer {
-	explainer, err := makeExplainer(modelSpec)
-	if err != nil {
-		klog.Fatal(err)
-	}
-
-	return explainer
-}
-
-func makeExplainer(explainerSpec *ExplainerSpec) (Explainer, error) {
+func getExplainer(explainerSpec *ExplainerSpec) (Explainer, error) {
```

Comment:
Can we log on line 90? We need to make sure that when we swallow the errors, it's debuggable from a kfserving-controller logs perspective. Make sure the predictor/transformer also log.

Reply:
Can we somehow update the status to help debugging?

Reply:
I'd love to have stronger error surfacing as well as terminal failures (I.e. malformed model). Can we punt this to v0.2+ scope?

Reply:
Sounds good. Will create issue.